templates/_helpers.tpl

```diff
@@ -64,14 +71,27 @@ Create the name of the service account to use
 {{/*
 Creates the fully qualified image tag needed
 */}}
-{{- define "timothy.image" -}}
+{{- define "timothy.webImage" -}}
```

Comment:
Good w/ this rename. Don't wanna cause confusion based on my comment about values.yaml structure.